modules/dxp/apps/analytics/analytics-reports-layout-impl/src/main/java/com/liferay/analytics/reports/layout/internal/info/item/LayoutAnalyticsReportsInfoItem.java

```diff
@@ -160,15 +199,15 @@ public boolean isShow(Layout layout) {
 		return true;
 	}
 
-	private ThemeDisplay _getThemeDisplay() {
+	private Optional<ThemeDisplay> _getThemeDisplayOptional() {
```

Comment:
would it be possible to use the directly the optional?

```java
return Optional.ofNullable(
			ServiceContextThreadLocal.getServiceContext()
		).map(
			ServiceContext::getThemeDisplay
		);
```

Reply:
Sure.